iroha_network/src/lib.rs

```diff
@@ -62,7 +62,7 @@ impl Network {
             .await
             .map_err(|e| e.to_string())?;
         stream.flush().await.map_err(|e| e.to_string())?;
-        let mut buffer = [0u8; BUFFER_SIZE];
+        let mut buffer = vec![0u8; BUFFER_SIZE];
```

Comment:
Also here why change to `vec`?

Reply:
Was made in alignment with API docs. We need to check that that approach will work even if size of transaction exceeds initial buffer size.